### PR TITLE
Textsans

### PR DESCRIPTION
**Pull request by mattosborn**
Going to wait for Chris for the updated font image as I have no photoshop and no idea of what fonts have been used in it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,12 @@ $fs-data: (
     18 22  // 6
 );
 
+$fs-textsans: (
+    12 16, // 1
+    13 18, // 2
+    14 22, // 3
```
Comment (kaelig) [inline README.md]: Extra comma is not needed

File: _font-scale.scss [hunk 2]
```diff
@@ -56,6 +57,12 @@ $fs-data: (
     18 22  // 6
 ) !default;
 
+$fs-textsans: (
+    12 16, // 1
+    13 18, // 2
+    14 22, // 3
```
Comment (kaelig) [inline _font-scale.scss]: Same here

**Comment by kaelig** [comment]
Awesome, thanks! Can you add a mention in the Changelog too?

**Comment by kaelig** [comment]
And if you could add 

```
$text-sans: "TextSans", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
```

to the readme as well.

**Comment by mattosborn** [comment]
Will leave this pull request open for now until the image is done.

**Comment by kaelig** [comment]
I've updated the PR with the new sizes and rewritten the history to make it look clean.

Merging this in :)

**Comment by mattosborn** [comment]
Nice one @kaelig ty.

**Comment by kaelig** [comment]
And thanks to @pearson82 for the new matrix!

**Pull request merged**